src/Sulu/Bundle/ContactBundle/Contact/AbstractContactManager.php

```diff
@@ -38,11 +37,18 @@
     public $em;
 
     /**
+     * @var string
+     */
+    protected $accountEntity;
+
+    /**
      * @param ObjectManager $em
+     * @param $accountEntityName
      */
-    public function __construct(ObjectManager $em)
+    public function __construct(ObjectManager $em, $accountEntityName)
```

Comment:
Is the `accountEntityName` really needed here? It's not used in this class, and set in the sub classes again.